COMMAND   PID USER   FD   TYPE DEVICE  SIZE/OFF    NODE NAME
simple1 67919 melo  cwd    DIR    1,1       374 5005484 /Users/melo/Dropbox/work/examples/zeromq/fork_context
simple1 67919 melo  txt    REG    1,1      8812 5024219 /Users/melo/Dropbox/work/examples/zeromq/fork_context/simple1
simple1 67919 melo  txt    REG    1,1    249952 3303454 /Users/melo/.homebrew/Cellar/zeromq/2.2.0/lib/libzmq.1.dylib
simple1 67919 melo  txt    REG    1,1    600576 3435227 /usr/lib/dyld
simple1 67919 melo  txt    REG    1,1 301068288 3751261 /private/var/db/dyld/dyld_shared_cache_x86_64
simple1 67919 melo    0u   CHR   16,4   0t62954    1707 /dev/ttys004
simple1 67919 melo    1u   CHR   16,4   0t62954    1707 /dev/ttys004
simple1 67919 melo    2u   CHR   16,4   0t62954    1707 /dev/ttys004

--- initial.lsof	2012-08-21 22:50:57.000000000 +0100
+++ ctx.lsof	2012-08-21 22:51:03.000000000 +0100
@@ -1,9 +1,19 @@
+simple1 67919 melo    3u    unix 0x77bbba67ea823b13       0t0         ->0x77bbba67edb7db73
+simple1 67919 melo    4u    unix 0x77bbba67edb7db73       0t0         ->0x77bbba67ea823b13
+simple1 67919 melo    5u    unix 0x77bbba67ea8234d3       0t0         ->0x77bbba67e868bb73
+simple1 67919 melo    6u    unix 0x77bbba67e868bb73       0t0         ->0x77bbba67ea8234d3
+simple1 67919 melo    7u  KQUEUE                                      count=0, state=0x2
+simple1 67919 melo    8u    unix 0x77bbba67ec7b3793       0t0         ->0x77bbba67edb8002b
+simple1 67919 melo    9u    unix 0x77bbba67edb8002b       0t0         ->0x77bbba67ec7b3793
+simple1 67919 melo   10u  KQUEUE                                      count=0, state=0x2
+simple1 67919 melo   11u    unix 0x77bbba67e8502fc3       0t0         ->0x77bbba67ec7b1533
+simple1 67919 melo   12u    unix 0x77bbba67ec7b1533       0t0         ->0x77bbba67e8502fc3

--- ctx.lsof	2012-08-21 22:51:03.000000000 +0100
+++ child.lsof	2012-08-21 22:51:27.000000000 +0100
@@ -1,19 +1,17 @@
 COMMAND   PID USER   FD     TYPE             DEVICE  SIZE/OFF    NODE NAME
-simple1 67919 melo    3u    unix 0x77bbba67ea823b13       0t0         ->0x77bbba67edb7db73
-simple1 67919 melo    4u    unix 0x77bbba67edb7db73       0t0         ->0x77bbba67ea823b13
-simple1 67919 melo    5u    unix 0x77bbba67ea8234d3       0t0         ->0x77bbba67e868bb73
-simple1 67919 melo    6u    unix 0x77bbba67e868bb73       0t0         ->0x77bbba67ea8234d3
-simple1 67919 melo    7u  KQUEUE                                      count=0, state=0x2
-simple1 67919 melo    8u    unix 0x77bbba67ec7b3793       0t0         ->0x77bbba67edb8002b
-simple1 67919 melo    9u    unix 0x77bbba67edb8002b       0t0         ->0x77bbba67ec7b3793
-simple1 67919 melo   10u  KQUEUE                                      count=0, state=0x2
-simple1 67919 melo   11u    unix 0x77bbba67e8502fc3       0t0         ->0x77bbba67ec7b1533
-simple1 67919 melo   12u    unix 0x77bbba67ec7b1533       0t0         ->0x77bbba67e8502fc3
+simple1 67955 melo    3u  unix 0x77bbba67ea823b13       0t0         ->0x77bbba67edb7db73
+simple1 67955 melo    4u  unix 0x77bbba67edb7db73       0t0         ->0x77bbba67ea823b13
+simple1 67955 melo    5u  unix 0x77bbba67ea8234d3       0t0         ->0x77bbba67e868bb73
+simple1 67955 melo    6u  unix 0x77bbba67e868bb73       0t0         ->0x77bbba67ea8234d3
+simple1 67955 melo    8u  unix 0x77bbba67ec7b3793       0t0         ->0x77bbba67edb8002b
+simple1 67955 melo    9u  unix 0x77bbba67edb8002b       0t0         ->0x77bbba67ec7b3793
+simple1 67955 melo   11u  unix 0x77bbba67e8502fc3       0t0         ->0x77bbba67ec7b1533
+simple1 67955 melo   12u  unix 0x77bbba67ec7b1533       0t0         ->0x77bbba67e8502fc3
